tests/cases/fourslash/server/semanticClassificationJs1.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference path="../fourslash.ts"/>
+
+//// @Filename: index.js
+//// 
+//// var Thing = 0;
+//// Thing.toExponential();
+
+// This test validates that an arbitrary JS file gets 
+// encoded semantic classifications when requested
+verify.encodedSemanticClassificationsLength("2020", 9)
```

Comment:
The above test is moved to the server subfolder, where it now goes through the same codepaths as vscode would